### PR TITLE
Updated README with more useful info

### PR DESCRIPTION
**Pull request by wadewegner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
+
 In order to develop on this project you will need to:  
-Download the repository into a local folder.  
-Locate this folder, run gradle_update.bat  
-This should download all dependencies and setup proper .classpath  
-Import the project into eclipse from the downloaded folder/repository.  
+
+1) Download the repository into a local folder.  
+
+2) Locate this folder, run `gradlew.bat` or `./gradlew`.
```
Comment (P3pp3rF1y) [inline README.md:8]: This should really be run gradlew setupDecompWorkspace

File: README.md [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
+
+This should download all dependencies and setup proper `.classpath`. Import the project into Eclipse from the downloaded folder/repository.
```
Comment (P3pp3rF1y) [inline README.md:10]: And this should be either run gradlew eclipse or gradlew idea depending on which IDE you use.

**Review by P3pp3rF1y — COMMENTED** [review]
I have avoided README for a long time so happy to see that you're taking a cut at it. the comments below are just to better clarify what needs to be done to prepare workspace. It's actually the same steps that need to be taken for most mods.

File: README.md (hunk 1 above)
Comment (wadewegner) [reply README.md:8]: `gradlew.bat setupDecompWorkspace`? I don't have Windows.

I'll update `./gradlew setupDecompWorkspace` and send a PR shortly.

Comment (P3pp3rF1y) [reply README.md:8]: you don't need to type ".bat" on windows. 
And somehow I thought you don't need "./" on mac when runing in current forlder in which case the command would be gradlew setupDecompWorkspace on both, but if you actually need to type that in that's the only difference.

**Pull request merged**